lib/overcommit/hook/pre_commit/execute_permissions.rb

```diff
@@ -2,6 +2,16 @@ module Overcommit::Hook::PreCommit
   # Checks for files with execute permissions, which are usually not necessary
   # in source code files (and are typically caused by a misconfigured editor
   # assigning incorrect default permissions).
+  #
+  # Protip: if you have some files that you want to allow execute permissions
```

Comment:
Isn't this just basic functionality available to every hook? Does it warrant a protip?